spec/indexers/collection_title_indexer_spec.rb

```diff
@@ -49,30 +24,18 @@
 
     context 'when related collections are provided' do
       let(:collections) { [collection] }
-
       let(:collection) do
-        Cocina::Models.build(
-          {
-            externalIdentifier: mock_rel_druid,
-            type: Cocina::Models::ObjectType.collection,
-            version: 1,
-            label: 'testing',
-            administrative: {
-              hasAdminPolicy: apo_id
-            },
-            access: {},
-            description: {
-              title: [{ value: 'Test object' }],
-              purl: "https://purl.stanford.edu/#{mock_rel_druid.delete_prefix('druid:')}"
-            },
-            identification: { sourceId: 'sul:1234' }
+        build(:collection, id: collection_druid).new(
```

Comment:
Can you just pass `title: 'Collection test object'` to `build` instead of using `.new(description: ...)`?